src/main/java/com/wavefront/opentracing/TracerParameters.java

```diff
@@ -104,18 +106,47 @@ static Integer toInteger(String value) {
     return integer;
   }
 
-  static Map<String, String> toCustomTags(String value, String delimiter) {
-    String[] components = value.split(delimiter);
-    if (components.length % 2 == 0) {
-      Map<String, String> customTags = new HashMap<>();
-      for (int i = 0; i < components.length; i += 2) {
-        customTags.put(components[i], components[i + 1]);
-      }
-      return customTags;
-    } else {
-      logger.log(Level.WARNING, "Failed to convert Tracer parameter value '" + value +
-          "' to custom tags -- key exists without value");
+  static Map<String, String> toCustomTags(Map<String, String> params) {
```

Comment:
mark the method @Nullable if you are returning null?

Reply:
fixed